docs/email/components.html

```diff
@@ -0,0 +1,528 @@
+    <li style="margin: 0 0 10px 30px;">You get a list item.</li>
+    <li style="margin: 0 0 10px 30px;">And you get a list item!</li>
+    <li style="margin: 0 0 0 30px;">Everyone gets a list item!</li>
+</ol>
+{% endhighlight %}
+        <div class="stacks-preview--example" style="font-family: arial, sans-serif; font-size: 15px; line-height: 19px; color: #54595F; text-align: left;">
+            <ul style="padding: 0; margin: 0 0 15px 0; list-style-type: disc;">
+                <li style="margin: 0 0 10px 30px;">You get a list item.</li>
+                <li style="margin: 0 0 10px 30px;">And you get a list item!</li>
+                <li style="margin: 0 0 0 30px;">Everyone gets a list item!</li>
+            </ul>
+            <ol style="padding: 0; margin: 0 0 15px 0; list-style-type: decimal;">
+                <li style="margin: 0 0 10px 30px;">You get a list item.</li>
+                <li style="margin: 0 0 10px 30px;">And you get a list item!</li>
+                <li style="margin: 0 0 0 30px;">Everyone gets a list item!</li>
+            </ol>
+        </div>
+    </div>
+    {% header {element: 'h3', text: 'Links'} %}
+    <div class="stacks-preview">
+{% highlight html linenos %}
+<a href="https://stackoverflow.com/" style="color: #0077CC; text-decoration: none;">Go to the homepage</a>
+{% endhighlight %}
+        <div class="stacks-preview--example" style="font-family: arial, sans-serif; font-size: 15px; line-height: 19px; color: #54595F; text-align: left;">
+            <a href="https://stackoverflow.com/" style="color: #0077CC; text-decoration: none;">Go to the homepage</a>
+        </div>
+    </div>
+    {% header {element: 'h3', text: 'Links in Markdown'} %}
+    <p class="stacks-copy">
+        If email content is written in markdown, we can't write inline CSS into link tags. Left unstyled, <a href="#" style="color: blue; text-decoration: underline;">links will appear like this</a>. To style links contained in markdown text, add a <code class="stacks-code">class="has-markdown"</code> to any parent tag that might contain a markdown link.
+    </p>
+    <div class="stacks-preview">
+{% highlight html linenos %}
+<p style="margin: 0 0 15px 0;">@_s("I'm a text string in markdown.")</p>
+<p style="margin: 0 0 15px 0;" class="has-markdown">@_m("I'm a *bold* [text link](https://www.stackoverflow.com) in markdown.")</p>
+{% endhighlight %}
+        <div class="stacks-preview--example" style="font-family: arial, sans-serif; font-size: 15px; line-height: 19px; color: #54595F; text-align: left;">
+            <p style="margin: 0 0 15px 0;">I'm a text string in markdown.</p>
+            <p style="margin: 0;">I'm a <strong>bold</strong> <a href="https://www.stackoverflow.com" style="color: #0077CC; text-decoration: none;">text link</a> in markdown.</p>
+        </div>
+        <aside class="s-notice s-notice__info s-notice__outlined mt24" role="status" aria-hidden="false">
```

Comment:
Move the notice outside of `div.stacks-preview--example`. It's getting an odd box-shadow connection.

![image](https://user-images.githubusercontent.com/1544281/43609629-9237e094-9672-11e8-991d-b82b806dddc6.png)